doc/semitrans.xml

```diff
@@ -330,6 +330,41 @@ true]]></Example>
   </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DirectProduct">
+<ManSection>
+  <Func Name = "DirectProduct" Arg = "S[, T, ...]"/>
+  <Oper Name = "DirectProductOp" Arg = "list, S"/>
+  <Returns>A transformation semigroup.</Returns>
+  <Description>
+    <C>DirectProduct</C> takes an arbitrary positive number of transformation
+    semigroups and returns the <E>direct product</E> of those semigroups, which
```

Comment:
the `<E>direct product</E>` -> a transformation semigroup isomorphic to the direct product of these semigroups. In the package we tend to use `<E>` tags when we're defining a concept rather than just stating a concept.